api/src/test/java/uk/gov/dhsc/htbhf/claimant/entitlement/CycleEntitlementCalculatorIntegrationTests.java

```diff
@@ -30,12 +30,12 @@ void shouldCalculateCorrectEntitlementForNoChanges() {
         LocalDate fourthBirthdayInOneYear = LocalDate.now().minusYears(3);
         List<LocalDate> childrenDatesOfBirth = asList(firstBirthdayInSixMonths, fourthBirthdayInOneYear);
 
-        int entitlementAmount = cycleEntitlementCalculator.calculateEntitlementInPence(claimant, childrenDatesOfBirth);
+        CycleVoucherEntitlement cycleVoucherEntitlement = cycleEntitlementCalculator.calculateCycleVoucherEntitlement(claimant, childrenDatesOfBirth);
 
         // Calculation per week: pregnant = 0 vouchers, children under 1 = 2 vouchers, children under 4 = 1 voucher
         // Total per week = 3 vouchers. Total over 4 weeks = 12 vouchers
         // Voucher value = 310 pence. Total voucher value for cycle = 3720 pence
-        assertThat(entitlementAmount).isEqualTo(3720);
+        assertThat(cycleVoucherEntitlement.getTotalVoucherValueInPence()).isEqualTo(3720);
```

Comment:
Would it be worthwhile adding assertions for the individual vouchers in these tests too?

Reply:
Done.